fe/fe-core/src/main/java/com/starrocks/alter/RollupJobV2.java

```diff
@@ -827,24 +823,16 @@ public void gsonPostProcess() throws IOException {
         // parse the define stmt to schema
         SqlParser parser = new SqlParser(new SqlScanner(new StringReader(origStmt.originStmt),
                 SqlModeHelper.MODE_DEFAULT));
-        ConnectContext connectContext = new ConnectContext();
-        connectContext.setCluster(SystemInfoService.DEFAULT_CLUSTER);
-        Database db = GlobalStateMgr.getCurrentState().getDb(dbId);
-        if (db == null) {
-            return;
-        }
-        connectContext.setDatabase(db.getFullName());
-        Analyzer analyzer = new Analyzer(GlobalStateMgr.getCurrentState(), connectContext);
-        analyzer.setIgnoreCast();
         CreateMaterializedViewStmt stmt = null;
         try {
             stmt = (CreateMaterializedViewStmt) SqlParserUtils.getStmt(parser, origStmt.idx);
             stmt.setIsReplay(true);
-            stmt.analyze(analyzer);
```

Comment:
why  needn't analyze?

Reply:
Because the main purpose of this code is to restore defineExpr.
The old idea is to use Analyzer to generate Stmt and then calculate MVColumn, and generate defineExpr through MVColumn, but our new Analyzer will use this method again, because the new one uses QueryStatement, and there is no old SelectStmt processing.
In addition, there is another advantage of not analyzing, that is, when the table name changes, only parse does not actually affect the logic of restoring defineExpr. And if you use Analyze, it will be easy to report errors here. resulting in other problems.